yamale/validators/validators.py

```diff
@@ -157,6 +157,31 @@ def _is_valid(self, value):
         return True
 
 
+class All(Validator):
+    """All validators must succeed validator"""
+
+    tag = "all"
+
+    def __init__(self, *args, **kwargs):
+        self.validators = [val for val in args if isinstance(val, Validator)]
+        super(All, self).__init__(*args, **kwargs)
+
+    def _is_valid(self, value):
+        return True
+
+    def is_valid(self, value):
```

Comment:
It seems perhaps better to override the `validate` method (which is called by `is_valid`) instead, along the lines of

```py
def validate(self, value):
    errors = []
    for validator in self.validators:
        errors += validator.validate(value)
    return errors
```

As it stands, `all_instance.is_valid(value)` can return `False` while `all_instance.validate(value)` returns no errors, which seems inconsistent.